topics/releases.md

```diff
@@ -1,144 +1,98 @@
-[Valkey GitHub repository](https://github.com/valkey-io/valkey).
-
-This branch is the source tree where most of the new features are under
-development. `unstable` is not considered production-ready: it may contain
-critical bugs, incomplete features, and is potentially unstable.
-
-However, we try hard to make sure that even the unstable branch is usable most
-of the time in a development environment without significant issues.
-
-### Release candidate
-
-New minor and major versions of Valkey begin by branching off the `unstable`
-branch. The branch name is the target release on the form *major.minor*.
-Subsequent patch releases are made on the same branch.
-
-For example, when Valkey 7.2.5 was released, the release was made on the `7.2`
-branch, which had been branched off from `unstable` earlier.
-
-Bug fixes and new features that can be stabilized during the release's time
-frame are committed to the unstable branch and backported to the release
-candidate branch. The `unstable` branch may include additional work that is not
-a part of the release candidate and scheduled for future releases.
-
-The first release candidate, or RC1, is released once it can be used for
-development purposes and for testing the new version. At this stage, most of
-the new features and changes the new version brings are ready for review, and
-the release's purpose is collecting the public's feedback.
-
-Subsequent release candidates are released every three weeks or so, primarily
-for fixing bugs. These may also add new features and introduce changes, but at
-a decreasing rate and decreasing potential risk towards the final release
-candidate.
-
-### Stable tree
-
-Once development has ended and the frequency of critical bug reports for the
-release candidate wanes, it is ready for the final release. At this point, the
-release is marked as stable and is released with "0" as its patch-level
-version.
+All Valkey releases are published in the [Valkey GitHub repository](https://github.com/valkey-io/valkey/releases).
 
 ## Versioning
 
-Stable releases liberally follow the usual `major.minor.patch` semantic
-versioning schema. The primary goal is to provide explicit guarantees regarding
-backward compatibility.
+Valkey stable releases will follow generally `major.minor.patch` [semantic versioning schema](https://semver.org/).
+We follow semantic versioning to provide explicit guarantees regarding backward compatibility.
 
-### Patch-Level versions
+### Patch versions
 
-Patches primarily consist of bug fixes and very rarely introduce any
-compatibility issues.
+PATCH versions are released with backwards compatible bug fixes and should not introduce new features.
 
-Upgrading from a previous patch-level version is almost always safe and
-seamless.
+Upgrading from a previous patch version should be safe and seamless.
+It should be safe to run a Valkey cluster with servers running on different patch versions.
```

Comment:
Speaking of cluster, we can also mention replication from/to?

Reply:
I didn't follow this comment.

Reply:
I think @zuiderkwast is saying we support replicating from the lower version of Valkey/Redis to a higher version but no guarantee if you replicate from a higher version of Valkey to a lower version (because RDB version might get bumped - separate discussion).

Reply:
Ok, let me try to add a sentence to address this.

Reply:
I meant just primary-replica of different patch versions, not necessarily using cluster.

```suggestion
It should be safe to run a primary-replica pair or a Valkey cluster with servers running on different patch versions.
```

Reply:
I understand now! I can add that.